tests/test_cli/test_run.py

```diff
@@ -175,8 +175,7 @@ def test_run(password_or_none):
 
 
 @pytest.mark.flaky(reruns=MAX_FLAKY_RERUNS)  # flaky on Windows
-@pytest.mark.skipif(
-    sys.version_info < (3, 7),
+@pytest.mark.skip(
```

Comment:
I'm marking this one as skip for newer version also. Viraj has looked into it and thinks it could be a pytest issue and that he'll try to fix it  when pinning the new pytest version on the py3.10 PR

Reply:
Ok, please open an issue for it.